### PR TITLE
i18n(it): translate guides/deploy/zerops.mdx

### PR DESCRIPTION
**Pull request by vclemenzi**
#### Description (required)

translate `guides/configuring-astro`

#### Related issues & labels (optional)

- Suggested label: i18n

### CODE REVIEW

File: src/content/docs/it/guides/deploy/zerops.mdx [hunk 1]
```diff
@@ -0,0 +1,164 @@
+---
+title: Effetua il Deploy del tuo sito Astro su Zerops
```
Comment (casungo) [inline src/content/docs/it/guides/deploy/zerops.mdx:2]:
```suggestion
title: Effettua il deploy del tuo sito Astro su Zerops
```

File: src/content/docs/it/guides/deploy/zerops.mdx [hunk 2]
```diff
@@ -0,0 +1,164 @@
...
+description: Come effetuare il deploy del tuo sito Astro usando Zerops.
```
Comment (casungo) [inline src/content/docs/it/guides/deploy/zerops.mdx:3]:
```suggestion
description: Come effettuare il deploy del tuo sito Astro usando Zerops.
```

File: src/content/docs/it/guides/deploy/zerops.mdx [hunk 3]
```diff
@@ -0,0 +1,164 @@
...
+type: deploy
+i18nReady: true
+---
+import PackageManagerTabs from '~/components/tabs/PackageManagerTabs.astro'
+import { Steps } from '@astrojs/starlight/components';
+
+[Zerops](https://zerops.io/) è una piattaforma cloud oriantata a gli sviluppatori che può essere usata per eseguire il deploy del tuo sito SSR Astro.
```
Comment (casungo) [inline src/content/docs/it/guides/deploy/zerops.mdx:10]:
```suggestion
[Zerops](https://zerops.io/) è una piattaforma cloud orientata agli sviluppatori che può essere usata per eseguire il deploy del tuo sito SSR Astro.
```

File: src/content/docs/it/guides/deploy/zerops.mdx [hunk 4]
```diff
@@ -0,0 +1,164 @@
...
+
+Questa guida ti accompagnerà nel processo di deployment di un sito Astro su Zerops utilizzando l'adattatore specifico per Node.js.
+
+## Prerequisiti
+
+- **Un progetto Astro che usa [l'adattarore `@astrojs/node` per il SSR](/it/guides/integrations-guide/node/)**
```
Comment (casungo) [inline src/content/docs/it/guides/deploy/zerops.mdx:16]:
```suggestion
- **Un progetto Astro che usa [l'adattatore `@astrojs/node` per il SSR](/it/guides/integrations-guide/node/)** 
```

File: src/content/docs/it/guides/deploy/zerops.mdx [hunk 5]
```diff
@@ -0,0 +1,164 @@
...
+import { Steps } from '@astrojs/starlight/components';
+
+[Zerops](https://zerops.io/) è una piattaforma cloud oriantata a gli sviluppatori che può essere usata per eseguire il deploy del tuo sito SSR Astro.
+
+Questa guida ti accompagnerà nel processo di deployment di un sito Astro su Zerops utilizzando l'adattatore specifico per Node.js.
+
+## Prerequisiti
+
+- **Un progetto Astro che usa [l'adattarore `@astrojs/node` per il SSR](/it/guides/integrations-guide/node/)** 
+- **Un account Zerops** - Se non ne hai gia uno, puoi [creare il tuo account Zerops](https://zerops.io/) gratuitamente.
```
Comment (casungo) [inline src/content/docs/it/guides/deploy/zerops.mdx:17]:
```suggestion
- **Un account Zerops** - Se non ne hai già uno, puoi [creare il tuo account Zerops](https://zerops.io/) gratuitamente.
```

File: src/content/docs/it/guides/deploy/zerops.mdx [hunk 6]
```diff
@@ -0,0 +1,164 @@
...
+
+:::tip[Iniziamo da un template]
+Il [Zerops per Astro - esempio per Node.js](https://github.com/zeropsio/recipe-astro-nodejs) può essere importato direttamente nella tua [Dashboard di Zerops](https://app.zerops.io/dashboard/projects), and deployed in one click!
```
Comment (casungo) [inline src/content/docs/it/guides/deploy/zerops.mdx:20]:
```suggestion
Il [Zerops per Astro - esempio per Node.js](https://github.com/zeropsio/recipe-astro-nodejs) può essere importato direttamente nella tua [Dashboard di Zerops](https://app.zerops.io/dashboard/projects), e distribuito con un clic!
```

File: src/content/docs/it/guides/deploy/zerops.mdx [hunk 7]
```diff
@@ -0,0 +1,164 @@
...
+      - port: 4321
+        httpSupport: true
+    enableSubdomainAccess: true
+    minContainers: 1
+```
+:::
+
+## Creare un progetto Zerops con Node.js
+
+Puoi creare un servizio Node.js per il tuo sito Astro attraverso il [Zerops `project add` wizard](https://app.zerops.io/dashboard/project-add), o importanto un sito Astro usando il `.yaml`.
```
Comment (casungo) [inline src/content/docs/it/guides/deploy/zerops.mdx:39]:
```suggestion
Puoi creare un servizio Node.js per il tuo sito Astro attraverso il [Zerops `project add` wizard](https://app.zerops.io/dashboard/project-add), o importando un sito Astro usando il `.yaml`.
```

File: src/content/docs/it/guides/deploy/zerops.mdx [hunk 8]
```diff
@@ -0,0 +1,164 @@
...
+
+La struttura dello YAML sottostante servira per creare un progetto chiamato `my-astro-sites` con Node.js v20 con l'hostname `hellothere`. Un progetto Zerops può contenere più siti Astro.
```
Comment (casungo) [inline src/content/docs/it/guides/deploy/zerops.mdx:41]:
```suggestion
La struttura dello YAML sottostante servirà per creare un progetto chiamato `my-astro-sites` con Node.js v20 con l'hostname `hellothere`. Un progetto Zerops può contenere più siti Astro.
```

File: src/content/docs/it/guides/deploy/zerops.mdx [hunk 9]
```diff
@@ -0,0 +1,164 @@
...
+            start: node dist/server/entry.mjs
+          envVariables:
+            HOST: 0.0.0.0
+            NODE_ENV: production
+    ```
+    </Fragment>
+  </PackageManagerTabs>
+
+### Attivare il processo di build e deploy usando GitHub / GitLab
+Per configurare il processo di build e deploy quando si fa un push a un branch o si crea una nuova release, vai al tuo servizio di Node.js e connetti il servizio di Zerops a una repositoriy GitHub o GitLab.
```
Comment (casungo) [inline src/content/docs/it/guides/deploy/zerops.mdx:135]:
```suggestion
Per configurare il processo di build e deploy quando si fa un push a un branch o si crea una nuova release, vai al tuo servizio di Node.js e connetti il servizio di Zerops a una repository GitHub o GitLab.
```

File: src/content/docs/it/guides/deploy/zerops.mdx [hunk 10]
```diff
@@ -0,0 +1,164 @@
...
+1. Installa la CLI di Zerops.
+		```shell
+		# Per scaricare il binario zcli direttamente,
+		# usa https://github.com/zeropsio/zcli/releases
+		npm i -g @zerops/zcli
+		```
+
+2. Vai su [`Settings > Access Token Management`](https://app.zerops.io/settings/token-management) nell'app di Zerops e genera un nuovo token per accedere.
+
+3. Esegui il log in usando il tuo access token usando il comando seguente:
```
Comment (casungo) [inline src/content/docs/it/guides/deploy/zerops.mdx:150]:
```suggestion
3. Effettua il login usando il tuo access token usando il comando seguente:
```

**Review by casungo — CHANGES_REQUESTED** [review]

**Review by casungo — APPROVED** [review]
LGTM!

**Review by yanthomasdev — APPROVED** [review]
LGTM! Good work peeps 💜

**Pull request merged**